pkg/apis/storage/v1alpha/local_storage_spec_test.go

```diff
@@ -0,0 +1,64 @@
+//
+// DISCLAIMER
+//
+// Copyright 2018 ArangoDB GmbH, Cologne, Germany
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Copyright holder is ArangoDB GmbH, Cologne, Germany
+//
+// Author Jan Christoph Uhde
+//
+
+package v1alpha
+
+import (
+	"testing"
+
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_LocalStorageSpec_Creation(t *testing.T) {
+	var (
+		class StorageClassSpec
+		local LocalStorageSpec
+		err   error
+	)
+
+	class = StorageClassSpec{"SpecName", true}
+	local = LocalStorageSpec{StorageClass: class, LocalPath: []string{""}}
+	err = local.Validate()
```

Comment:
Don't understand this and the next line.
Probably you want to use `assert.Error` or `assert.NoError` see https://godoc.org/github.com/stretchr/testify/assert#Error